totalRP3/Modules/Register/Characters/RegisterUICharacteristics.xml

```diff
@@ -53,6 +53,17 @@ https://raw.githubusercontent.com/Meorawr/wow-ui-schema/main/UI.xsd">
 				</Anchors>
 			</Button>
 		</Frames>
+		<Layers>
+			<Layer level="OVERLAY">
+				<FontString name="$parentPresetNameText" parentKey="PresetName" text="[preset]" inherits="GameFontNormal" justifyH="LEFT">
+					<Size x="100" y="18"/>
+					<Anchors>
+						<Anchor point="LEFT" x="15" y="0" relativePoint="RIGHT" relativeTo="$parentIcon"/>
+					</Anchors>
+					<Color r="1.0" g="1.0" b="1.0"/>
```

Comment:
```suggestion
				<FontString parentKey="PresetName" text="[preset]" inherits="GameFontHighlight" justifyH="LEFT">
					<Size x="100" y="18"/>
					<Anchors>
						<Anchor point="LEFT" x="15" y="0" relativePoint="RIGHT" relativeTo="$parentIcon"/>
					</Anchors>
```
With the above we don't need a name anymore, and we can remove the colour if we just use GameFontHighlight.